deploy/example/inline/nginx-pod-azuredisk-inline.yaml

```diff
@@ -0,0 +1,26 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: nginx-azuredisk-inline
+spec:
+  nodeSelector:
+    beta.kubernetes.io/os: linux
+  containers:
+  - image: nginx
+    name: nginx-azuredisk
+    command:
+    - "/bin/sh"
+    - "-c"
+    - while true; do echo $(date) >> /mnt/azuredisk/outfile; sleep 1; done
+    volumeMounts:
+    - name: azuredisk02
+      mountPath: "/mnt/azuredisk"
+  volumes:
+  - name: azuredisk02
+    csi:
+      driver: disk.csi.azure.com
+      volumeAttributes:
+        capacity: 10Gi # default is 1Gi
```

Comment:
only add two fields(`diskURI`, `fsType`) in `volumeAttributes`, and then we could create a new volumeid (exactly same as `diskURI`), this volumeid would be used in both node publish and node unpublish?

BTW, is it possible add a `volumeHandle` field like following, then we could have exact same logic as pv/pvc static provisioning:
```
  csi:
    driver: disk.csi.azure.com
    readOnly: false
    volumeHandle: /subscriptions/{sub-id}/resourcegroups/{group-name}/providers/microsoft.compute/disks/{disk-id}
    volumeAttributes:
      fsType: ext4
```